code/controllers/subsystem/shuttle.dm

```diff
@@ -178,9 +178,30 @@ SUBSYSTEM_DEF(shuttle)
 			Good luck.")
 			return
 		emergency = backup_shuttle
-	var/srd = CONFIG_GET(number/shuttle_refuel_delay)
-	if(world.time - SSticker.round_start_time < srd)
-		to_chat(user, "The emergency shuttle is refueling. Please wait [DisplayTimeText(srd - (world.time - SSticker.round_start_time))] before trying again.")
+	var/refuel_time = CONFIG_GET(number/shuttle_refuel_min_delay) * 2
+	// condensed names for readability of the actual equation
+	var/aa = 0 // amt_of_crew
+	var/xx = 0 // dead_crew
```

Comment:
\>commenting what the variable is with something short enough it would be better as the variable name itself